### PR TITLE
updating deploy setup

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,23 +115,65 @@ You can generate the salts on the [WordPress Salt Generator](https://api.wordpre
...
+
 - `SLACK_WEBHOOK` - webhook for the Slack notifications (ask your TL for this secret).
 - `BUGSNAG_API_KEY` - API key for the Bugsnag notifications (ask your TL for this secret).
 
 ### Adding custom secrets in the deploy process
 
-To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action.
+_Responsible person: DevOps or TL + Developer_
+
+To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapper in single quotes.
```
Comment (iobrado) [inline README.md]:
```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapped in single quotes.
```

Typo?

File: README.md [hunk 2]
```diff
@@ -108,23 +115,65 @@ You can generate the salts on the [WordPress Salt Generator](https://api.wordpre
...
 ```
 
-## Server setup
+## Env secrets/variables vs Repository secrets/variables
+
+GitHub offers two types of secrets: variables and secrets. The difference between them is that variables are visible in the logs and in the GUI, while secrets are not. Once set secret can't be viewed again.
+
+Also GitHub offers two types of usage for secrets: repository secrets and environment secrets. The difference between them is that repository secrets are available to all workflows in the repository, while environment secrets are available only to the workflows in the environment and this environment is defined in the workflow file.
+
+When setting up secrets, make sure to use environment secrets make sure you add them to the correct location and correct type based on the sensitivity of the data.
```
Comment (iobrado) [inline README.md]: This is a bit hard to read. Possible typo or missing comma?

**Review by iobrado — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dadadavorin) [inline README.md]:
```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind that the value is always wrapped in single quotes.
```

File: README.md (hunk 2 above)
Comment (dadadavorin) [inline README.md]:
```suggestion
When setting up secrets, make sure to use environment secrets and make sure you add them to the correct location and correct type based on the sensitivity of the data.
```

File: README.md [hunk 3]
```diff
@@ -108,23 +115,65 @@ You can generate the salts on the [WordPress Salt Generator](https://api.wordpre
...
+
 - `SLACK_WEBHOOK` - webhook for the Slack notifications (ask your TL for this secret).
 - `BUGSNAG_API_KEY` - API key for the Bugsnag notifications (ask your TL for this secret).
 
 ### Adding custom secrets in the deploy process
 
-To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action.
+_Responsible person: DevOps or TL + Developer_
+
+To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind you that the value is always wrapped in single quotes.
```
Comment (dadadavorin) [inline README.md]:
```suggestion
To add custom secrets in the deploy process add a custom step after the `Setup WordPress` step in the deploy action. Keep in mind that the value is always wrapped in single quotes.
```

**Review by piqusy — DISMISSED** [review]

**Review by dadadavorin — DISMISSED** [review]
Looks good. Just a few typos.

**Review by iobrado — DISMISSED** [review]

**Pull request merged**